modules/build-run/pages/sca-user-flows.adoc

```diff
@@ -0,0 +1,85 @@
+= Build and run an Application
+:experimental: // activate the 'menu' macro
+:description: User flows possible in Application mode
+
+== User flows in Application mode
+
+=== Creating a new project in Studio using Application mode
+
+[WARNING]
+====
+In order to generate the docker image, it's mandatory to have a maven repository working to get all the dependencies, including the latest Application runtime version.
+====
+
+. Run Bonita Studio {bonitaVersion} and create a new project.
+
+. Configure the project environment
+
+. Validate the project
+.. To work with Application mode, you need to explicitly mapping a form for each interaction form (instantiation and manual task)
+
+. Admin and user app is now available in extension, if you want to deploy it on the final bundle with Application mode, you need to add it manually
+
+.. Go to the Extension view
+
+.. Open Marketplace and filter on “Application type”
+
+.. You can now add the default application as you wish on the final bundle
+
+. At the root of the project folder, run `mvn clean package -Pbundle,docker "-Dbonita.environment=production"` 
+
+. Find the docker image name with `docker images` to be sure it's being created.
+
+. Run the docker image with command (adapting licenses folder if needed) `docker run --rm --name MyProjectName -h MyProjectName -e -v ~/lic/:/opt/bonita_lic/ -d -p 9000:8080 MyProjectNameImage-production:0.0.1`
```

Comment:
In case you mention a command like this, please use a 'source' section suche as:
[source,shell]
----
docker run ...
----
This is more readable and easier to copy the command with a single click